cmd/virt-launcher/libvirtd.sh

```diff
@@ -19,23 +19,6 @@
 
 set -xe
 
-# HACK
-# Use hosts's /dev to see new devices and allow macvtap
-mkdir -p /dev.container && {
-    mount --rbind /dev /dev.container
-    mount --rbind /host-dev /dev
-
-    # Keep some devices from the containerinal /dev
-    keep() { mount --rbind /dev.container/$1 /dev/$1; }
-    keep shm || :
-    keep mqueue
-    # Keep ptmx/pts for pty creation
-    keep pts
-    mount --rbind /dev/pts/ptmx /dev/ptmx
```

Comment:
All the keeps above were required to run the container on other OSses which were != fedora/centos.

I'd rather keep them.

Reply:
I agree they are needed, but you were keeping the devices from within the container. Now we don't overwrite the container devices anymore, therefore we don't need any extra "keep" logic.